### ISSUE TITLE
[FEATURE] Add issue/PR templates

### ISSUE THREAD

**Issue by Panquesito7** [TheAlgorithms/Solidity#30]
**Description**

Title says it all. :)
We can take the references from https://github.com/dev-protocol/devprotocol.xyz/tree/main/.github/ISSUE_TEMPLATE

What do you think about this? Let me know. 🙂

**Comment by mkubdev** [comment]
Agreed! Also, we should write requirements about adding Foundry test file in the PR. 🙂

**Comment by eshaanagarwal** [comment]
hi @Panquesito7 i would love to work on this. I just neede some references for writing the requirements on Foundry test file. Can you please give directions for that.
